hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/handlers/ContractCallHandlerTest.java

```diff
@@ -210,4 +214,14 @@ private TransactionBody contractCallTransactionWithInsufficientGas() {
                         .contractID(targetContract))
                 .build();
     }
+
+    private TransactionBody contractCallTransactionWithInsufficientGas() {
+        final var transactionID = TransactionID.newBuilder().accountID(payer).transactionValidStart(consensusTimestamp);
+        return TransactionBody.newBuilder()
+                .transactionID(transactionID)
+                .contractCall(ContractCallTransactionBody.newBuilder()
+                        .gas(INTRINSIC_GAS_FOR_0_ARG_METHOD - 1)
+                        .contractID(invalidContract))
```

Comment:
I think here and L214 are swapped given the method names!

Reply:
yep caught that. fixed in update